steely/plugins/roomcheck.py

```diff
@@ -74,6 +77,8 @@ def academic_week_number(date):
 
 def academic_hour(date):
     hour = date.hour
+    if hour < 8 or hour > 21:
+        raise ValueError
```

Comment:
raise without message looks funny,  
you might better doing something like `ValueError("invalid academic hour")` here, and keep above,  
_or_
`ValueError("building is closed at this time")` here, and when you catch it above `as exc`, do `send_message(str(exc))`  

not sure what's best, that's up to you